xml/System.Windows.Forms/ToolStripItem.xml

```diff
@@ -7294,8 +7297,12 @@ If you set the <xref:System.Windows.Forms.ToolStripItem.ImageKey%2A> property, t
         <ReturnType>System.EventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Occurs when value of the <see cref="P:System.Windows.Forms.ToolStripItem.Selected" /> property changes.</summary>
+        <remarks>
+          <para>This event is raised when item is selected by mouse, keyboard or programmatically.</para>
+          <para>.NET Core 3.1 had removed MainMenu and MenuItem controls. ToolStripMenuItem.SelectedChanged event</para>
+          <para>is recommended as a replacement for MenuItem.Select event.</para>
```

Comment:
```suggestion
          <para>is recommended as a replacement for the MenuItem.Select event.</para>
```

Reply:
This and the previous suggestion wasn't applied, looks like a bad merge...

Reply:
Thanks for the catch!